packages/meteor-accounts-saml/saml_client.js

```diff
@@ -16,22 +16,22 @@ if (!Accounts.saml) {
 
 const MeteorLogout = Meteor.logout;
 const logoutBehaviour = {
-	TERMINATE_SAML: "SAML",
-	ONLY_RC: "Local"
-}
+	TERMINATE_SAML: 'SAML',
+	ONLY_RC: 'Local'
+};
 
 Meteor.logout = function() {
 	const samlService = ServiceConfiguration.configurations.findOne({service: 'saml'});
 	if (samlService) {
 		const provider = samlService.clientConfig && samlService.clientConfig.provider;
 		if (provider && samlService.logoutBehaviour === logoutBehaviour.TERMINATE_SAML) {
 			if (samlService.idpSLORedirectURL) {
-				return Meteor.logoutWithSaml({ provider });
+				return Meteor.logoutWithSaml({provider});
 			}
 		}
-	}
-	if(samlService.logoutBehaviour === logoutBehaviour.ONLY_RC){
-		console.info("SAML session not terminated, only the Rocket.Chat session is going to be killed")
+		if (samlService.logoutBehaviour === logoutBehaviour.ONLY_RC) {
```

Comment:
this solves an issue logging out without saml.

It actually has got not much to do with this PR, but it's targeted develop as well and I came across it while testing

Reply:
I don't know how to test this